lib/fixpoint_test_helpers.rb

```diff
@@ -1,8 +1,8 @@
 # Helper methods to be included into RSpec
 module FixpointTestHelpers
-  def restore_fixpoint(fixname)
+  def restore_fixpoint(fixname, base: ActiveRecord::Base)
```

Comment:
nice to use a keyword argument here.
why not pass the connection directly? I feel that `base` is a little confusing/generic as a argument name

Reply:
Done!